src/main/java/core/basesyntax/strategy/BlackFridayDiscountService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy;
+
+public class BlackFridayDiscountService implements DiscountService {
+    private static final double DISCOUNT_PERCENT = 45.0;
```

Comment:
[Do not make amount constant](https://mate-academy.github.io/jv-program-common-mistakes/java-core/patterns/strategy#do-not-create-constants-for-the-discount-type-and-discount-amount)

Reply:
sorry overlooked